<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>439</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>HTTP接口测试</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>22</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>GET</string>
   </property>
   <item>
    <property name="text">
     <string>GET</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>POST</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PUT</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="urlEdit">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>22</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>21</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="respTextEdit">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>71</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="reqTextEdit">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>71</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>53</y>
     <width>150</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>                 请求报文</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>53</y>
     <width>144</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>                响应报文</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>375</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
